scripts/asm-installer/release_asm_installer

```diff
@@ -133,18 +143,28 @@ publish_script() {
   local BRANCH_NAME; BRANCH_NAME="${1}"
   local VERSION; VERSION="${2}"
   local SCRIPT_NAME; SCRIPT_NAME="${3}"
-  local FILE_NAME; FILE_NAME="${SCRIPT_NAME}_${VERSION}"
-  local FILE_PATH; FILE_PATH="${BUCKET_PATH}/${FILE_NAME}"
-  local FILE_URI; FILE_URI="${BUCKET_URL}/${FILE_PATH}"
+  local TAG;
```

Comment:
These happen to be git tags, but they're versions first. (i.e. we name the tag after the version, not the other way around)

Can we change TAG to STABLE_VERSION?

Reply:
Done